### DISCUSSION TITLE
Missing dependencies: lib32gcc-s1 lib32stdc++6 libsdl2-2.0-0:i386 steamcmd

### DISCUSSION THREAD

**Discussion by Soham-Das-afk** [GameServerManagers/LinuxGSM#4664] [Q&A]
Linux instance-20240922-1117 
6.5.0-1027-oracle 
aarch64 
GNU/Linux

ldd (Ubuntu GLIBC 2.35-0ubuntu3.8) 2.35

tmux 3.2a

Checking Palworld Dependencies
=================================
bc
binutils
bsdmainutils
bzip2
ca-certificates
cpio
curl
distro-info
file
gzip
hostname
jq
lib32gcc-s1
lib32stdc++6
libsdl2-2.0-0:i386
netcat
pigz
python3
steamcmd
tar
tmux
unzip
util-linux
uuid-runtime
wget
xz-utils
Warning! Missing dependencies: lib32gcc-s1 lib32stdc++6 libsdl2-2.0-0:i386 steamcmd

someone help

**Comment by TheInspector556** [comment]
I have the same issue here for Oracle servers

**Comment by PlayMTL** [comment]
You are running ARM arch, which doesn't support the needed packages.
